brdr/aligner.py

```diff
@@ -109,14 +109,15 @@ def __init__(
         # dictionary to store all thematic geometries to handle
         self.dict_thematic: dict[str, BaseGeometry] = {}
         # dictionary to store all unionedthematic geometries
+        self.dict_thematic_properties: dict[str, dict] = ({} )  # dictionary to store properties of the reference-features (optional)
+
         self.thematic_union = None  # to save a unioned geometry of all thematic polygons;
```

Comment:
als je comments boven de relevante lijn code plaatst, neemt dit minder horizontale plaats in. onze linter verwacht een maximale lijnbreedte van 88 tekens. 

lijn 112 zou ook onder 114 moeten komen